js/components/dataset/list.vue

```diff
@@ -80,9 +79,15 @@ export default {
         datasets: null,
         downloads: {
             type: Array,
-            default: function() {
+            default() {
```

Comment:
Sadly, you can't do that because it's already some kind of magic trickery (note that it's `default()`, not `default: ()`)